pkg/apis/apps/v1alpha1/types.go

```diff
@@ -0,0 +1,59 @@
+package v1alpha1
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type TemplateType string
+
+const (
+	StatefulSetTemplateType TemplateType = "StatefulSet"
+	DeploymentTemplateType  TemplateType = "Deployment"
+)
+
+// YurtAppSetConditionType indicates valid conditions type of a YurtAppSet.
+type YurtAppSetConditionType string
+
+const (
+	// PoolProvisioned means all the expected pools are provisioned and unexpected pools are deleted.
+	PoolProvisioned YurtAppSetConditionType = "PoolProvisioned"
+	// PoolUpdated means all the pools are updated.
+	PoolUpdated YurtAppSetConditionType = "PoolUpdated"
+	// PoolFailure is added to a YurtAppSet when one of its pools has failure during its own reconciling.
+	PoolFailure YurtAppSetConditionType = "PoolFailure"
+)
```

Comment:
line 16~25 should be defined in `yurtappset_type.go` file.

Reply:
This part of the code is directly copied from `yurt-app-manager` and is not used at present. I deleted this part directly.